configuration/README.md

```diff
@@ -37,35 +49,50 @@ FieldTags:
   Val: source
 ```
 
-Sinks and sanitizers are identified via regexp according to package, method, and (optional) receiver name.
+Sinks and sanitizers are identified by package, method, and (optional) receiver name.
+Again, these may be specified by either a provided string literal or regexp.
 
 ```yaml
 Sinks:
-- PackageRE: <package path regexp>
+- Package:  "literal/package/path" 
   ReceiverRE: <type name regexp>
   MethodRE: <method name regexp>
 Sanitizers:
 - PackageRE: <package path regexp>
   ReceiverRE: <type name regexp>
-  MethodRE: <method name regexp>
+  Method: "mySanitizer"
 ```
 
 Taint propagation is performed automatically and does not need to be explicitly configured.
 
-For matchers that accept a `ReceiverRE` regexp matcher, an unspecified string will match any (or no) receiver.
-To match only methods without any receiver (i.e., a top-level function), use the matcher `^$` to match an empty-string receiver name.
+### Important Note
+
+Configuration of the above matchers does not require listing of all arguments.
```

Comment:
I think it would be clearer to put the emphasis on the behavior: if a key is not specified, all values for that key will be matched. (As a consequence of that, you may not need to list some keys).

I think in the context of YAML, "key" or "key-value pair" may be clearer than "argument". WDYT? :thinking: